app/src/main/java/com/example/taskyy/ui/AgendaScreen.kt

```diff
@@ -86,23 +79,60 @@ fun AgendaScreen(state: AgendaState, navController: NavController, onEvent: (Age
         }
         Surface(
             modifier = Modifier
-                .fillMaxHeight(.85F)
+                .fillMaxHeight(.90F)
                 .fillMaxWidth(),
             shape = AbsoluteRoundedCornerShape(30.dp, 30.dp)
         ) {
             Column(
-                horizontalAlignment = Alignment.CenterHorizontally,
-                verticalArrangement = Arrangement.Top,
-                modifier = Modifier
-                    .fillMaxHeight(.75f)
-                    .padding(30.dp)
+                verticalArrangement = Arrangement.Bottom,
             ) {
-                Spacer(modifier = Modifier.height(10.dp))
+                //I know this is wrong but I don't know how to push this button to the bottom right of the screen
+                Spacer(modifier = Modifier.height(645.dp))
+                Row(
+                    horizontalArrangement = Arrangement.End,
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .padding(horizontal = 10.dp)
+                ) {
+                    AddAgendaItem(onEvent = onEvent)
+                }
```

Comment:
Such a button like the add button is called a floating action button. Best for that is to use a Scaffold:
```
Scaffold(
    floatingActionButton = {
        AddAgendaItem(...)
    }
) { padding -> 
    // Screen content
}
```